modules/distribution-zones/src/main/java/org/apache/ignite/internal/distributionzones/rebalance/RebalanceUtil.java

```diff
@@ -186,22 +193,22 @@ public static CompletableFuture<Void> updatePendingAssignmentsKeys(
                     LOG.info(
                             "Update metastore pending partitions key [key={}, partition={}, table={}/{}, newVal={}]",
                             partAssignmentsPendingKey.toString(), partNum, tableDescriptor.id(), tableDescriptor.name(),
-                            ByteUtils.fromBytes(partAssignmentsBytes));
+                            partAssignments);
 
                     break;
                 case PLANNED_KEY_UPDATED:
                     LOG.info(
                             "Update metastore planned partitions key [key={}, partition={}, table={}/{}, newVal={}]",
                             partAssignmentsPlannedKey, partNum, tableDescriptor.id(), tableDescriptor.name(),
-                            ByteUtils.fromBytes(partAssignmentsBytes)
+                            partAssignments
                     );
 
                     break;
                 case PLANNED_KEY_REMOVED_EQUALS_PENDING:
                     LOG.info(
                             "Remove planned key because current pending key has the same value [key={}, partition={}, table={}/{}, val={}]",
                             partAssignmentsPlannedKey.toString(), partNum, tableDescriptor.id(), tableDescriptor.name(),
-                            ByteUtils.fromBytes(partAssignmentsBytes)
+                            Assignments.fromBytes(partAssignmentsBytes)
```

Comment:
Why not `partAssignments`?

Reply:
You didn't understand me. I meant, that you replaced similar code with simple `partAssignments` reference instead of deserialization in all similar places but this one. Why?

Reply:
Oh, I guess I forgot to do it